### PR TITLE
Add rdw-kentekencheck extension

### PR DESCRIPTION
**Pull request by jeffreyvanhees**
## Description

Adds the ability to search the dutch license plate register by license plates or car brands and display available information about them. The data comes from the Dutch government's open API.

## Screencast
Screenshot 2024-05-31 at 14 00 33

## Checklist

- [X] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [X] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [X] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [X] I checked that files in the `assets` folder are used by the extension itself
- [X] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/rdw-kentekencheck/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "rdw-kentekencheck",
+  "title": "RDW Kentekencheck",
```
Comment (pernielsentikaer) [inline extensions/rdw-kentekencheck/package.json:4]: Can we come up with a english name like `Dutch License Plate (Search)`

**Review by pernielsentikaer — COMMENTED** [review]
- Could you add at least one [screenshot](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) of the extension

File: extensions/rdw-kentekencheck/package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "RDW Kentekencheck",
+      "subtitle": "Search details about Dutch cars by their license plate",
+      "description": "Shows details about Dutch cars by their license plate",
+      "mode": "view",
+      "icon": "command-icon.png"
```
Comment (pernielsentikaer) [inline extensions/rdw-kentekencheck/package.json:19]: You can reuse list-icon.png here 🙂

File: extensions/rdw-kentekencheck/package.json (hunk 1 above)
Comment (jeffreyvanhees) [reply extensions/rdw-kentekencheck/package.json]: See https://github.com/raycast/extensions/pull/12689/commits/483d0c34bf615e82eb8249f4cd24dbbf612e7b1f

File: extensions/rdw-kentekencheck/package.json (hunk 2 above)
Comment (jeffreyvanhees) [reply extensions/rdw-kentekencheck/package.json]: [Done!](https://github.com/raycast/extensions/pull/12689/commits/483d0c34bf615e82eb8249f4cd24dbbf612e7b1f)

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**